db.py

```diff
@@ -45,31 +45,26 @@ def get_all_users_following(userid):
   return followers
 
 
-def get_all_users_unfollowing(followers):
+def get_all_users_unfollowing(userid, followers):
   """
   Returns a list of users that the user in question is not following
   :param userid:
   :return: []
   """
+  # Get the list of users following the given user
+  cursor.execute('SELECT follower FROM followers WHERE followee = :userid', {'userid': userid})
 
-  followers_string = []
-  for item in followers:
-    followers_string.append(str(item))
+  # Above query returned a list of typle [(2,), (3,)] so access the index 0 to get the id
+  followers = [row[0] for row in cursor.fetchall()]
 
-  followers_comman_sep = ','.join(followers_string)
+  # Get the list of all users and exclude the userid inself since it can not follow himself
+  cursor.execute('SELECT _id FROM users')
```

Comment:
Looks pretty good! Great work!

Please change this to a model function call e.g. get_all_users()